### PR TITLE
Uzbekistan frequency plan

### PR DESCRIPTION
**Pull request by Jaime-Trinidad**
Frequency plan for Water meter projects in uzbekistan

#### Summary

This Uzbekistan frequency plan was asked for water meter projects, specifications were sent:

image

#### Changes

- 4 channels 926.2, 926.4, 926.6, 926.8
- 3 uplink channels 926.4, 926.6, 926.8
- RX2 in channel 926.2

#### Notes for Reviewers

...

#### Checklist

- [ ] Scope: The referenced issue is addressed, there are no unrelated changes.
- [ ] Compatibility: The changes are backwards compatible, they don't break existing deployments.
- [ ] Testing: The changes are tested.
- [ ] Documentation: Relevant documentation is added or updated.

### CODE REVIEW

File: frequency-plans.yml [hunk 1]
```diff
@@ -860,3 +860,10 @@
   base-id: AS_920_923
   country-codes: [sg]
   file: SG_920_923_TTN.yml
+
+- id: UZ_923_1
+  band-id: AS_923
+  name: Uzbekistan 915-928 MHz with only default channels
+  description: Compatibility frequency plan for Uzbekistan with common channels in the 926.2-926.8 MHz sub-band
+  base-frequency: 915
+  file: UZ_923_1.yml
```
Comment (johanstokking) [inline frequency-plans.yml:869]: Add `country-codes`

**Review by johanstokking — COMMENTED** [review]
> This Uzbekistan frequency plan was asked for water meter projects, specifications were sent [...]

Is this a particular project or is this making maximum use of the unlicensed spectrum in Uzbekistan?

**Comment by Jaime-Trinidad** [comment]
> > This Uzbekistan frequency plan was asked for water meter projects, specifications were sent [...]
> 
> Is this a particular project or is this making maximum use of the unlicensed spectrum in Uzbekistan?

Axioma is doing the water meter project and they gave them the frequency specification, they only can use those channels from AS_923 frequency

**Review by johanstokking — APPROVED** [review]

**Review by adriansmares — APPROVED** [review]

**Comment by Jaime-Trinidad** [comment]
@adriansmares  can you merge this PR?

**Pull request merged**